### PR TITLE
Umr0 video render

### PR DESCRIPTION
**Pull request by chenyanxzhu**

### CODE REVIEW

File: i915.c [hunk 1]
```diff
@@ -813,6 +819,14 @@ static int i915_init(struct driver *drv)
 		height = kDefaultCursorHeight;
 	i915->cursor_height = height;
 
+	drv_loge("-zcy- %s() dev_ver=%d dev_id=0x%x %s %s %s %s %s %s\n", __func__,
```
Comment (feijiang1) [inline i915.c:822]: please remove the -zcy

File: i915.c [hunk 2]
```diff
@@ -964,6 +978,10 @@ static int i915_bo_compute_metadata(struct bo *bo, uint32_t width, uint32_t heig
 		bo->meta.tiling = I915_TILING_4;
 		break;
 	}
+	char *pFormat = (char*)&format;
+	drv_loge("-zcy- %s() w/h=%d/%d format=%c%c%c%c use_flags=0x%x mod/cnt=0x%x/%d tiling=0x%x\n", __func__,
```
Comment (feijiang1) [inline i915.c:982]: please remove the -zcy

Comment (chenyanxzhu) [reply i915.c:982]: For debug purpose, will remove the log when test is ok

**Review by feijiang1 — APPROVED** [review]
LGTM

**Review by phreer — APPROVED** [review]
LGTM. Chenyan, please start to merge this PR, thanks!

**Review by feijiang1 — APPROVED** [review]
LGTM

**Review by phreer — APPROVED** [review]
LGTM.

**Review by feijiang1 — APPROVED** [review]
LGTM

**Pull request merged**